about.html

```diff
@@ -0,0 +1,150 @@
+             
+          </p>
+      </span>
+    </section>
+    <section class="logo-history">
+      <span>
+        <h2 class="logo-title">73rd FIFA Congress logo history</h2>
+        <hr class="line-1 line-11">
+      </span>
+        <p class="logo-details">73rd FIFA Congress logo was decided on 16 November 2022 at 09:00</p>
+        <div class="image-and-subtitle">
+            <img src="./images/page-images/73rd congress.png" alt="FIFA logo" class="logo-image">
+            <div class="more-description">
+              <hr>
+              <small class="image-description">73rd FIFA Congress 2023</small>
+              <hr>
+            </div>
+        </div>
+    </section>
+    <section class="past-events">
+      <span class="re-arrange">
+        <h2 class="event-title">See past FIFA Congress Meeting</h2>
+        <hr class="line-1 line-12">
+      </span>
+      <p class="past-event-description">Take a look at past 2 years FIFA Congress Meeting</p> 
+      <span class="past-events-photos">
+        <div class="event-image-1">
+          <h3 class="event-location">FIFA Congress Meeting at Doha</h3>
+          <p class="event-time">2022</p>
+        </div>
+        <div class="event-image-2">
+          <h3 class="event-location">FIFA Congress Meeting at Switzeland</h3>
+          <p class="event-time">2021</p>
+        </div>
+      </span>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-1-container">
+      <section class="footer-section-1" id="partners">
+        <span>
+          <h2 class="footer-title">Partners</h2>
+          <hr class="line-1 line-13">
+        </span>
+        <ul class="partners-list">
+          <li>
+            <a href="#"><img src="./images/page-images/coca.png" alt="Cocacola logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/adidas.png" alt="Adidas logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/Qatar-Airways.png" alt="Quatar Airways logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/hundai.png" alt="Hyundai logo" class="partners-logo"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/page-images/visa.png" alt="Visa logo" class="partners-logo"></a>
+          </li>
+        </ul>
+      </section>
+    </div>
+    <div class="footer-2-container">
+      <section class="footer-section-2">
+        <div class="headers">
+          <h2 class="copyright-name">FIFA</h2>
+          <h3 class="copyright-title">73rd FIFA Congress Meeting in Kigali</h3>
+        </div>
+        <div class="paragraphs">
+          <p class="copyrigh-description">FIFA Congress Meeting in Kigali</p>
+          <p class="allright">All Rights Reserved</p>
+        </div>
+      </section>
+    </div>
+  </footer>
```

Comment:
- [ ] On desktop version, the about footer section has a black background color and not a white background color. KIndly implement this:

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/229807880-d80dd636-76d6-4388-bc4b-35dfa0f27797.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/229807796-b358d691-7d4f-4b9a-8d4a-a2fa78854472.png)